### DISCUSSION TITLE
购买的两年专业版还未到期就已经变为免费版了2024.10.30

### DISCUSSION THREAD

**Discussion by Joah618** [getlantern/lantern-forum-cn#1008] [General]
购买的两年专业版还未到期就已经变为免费版了,邮箱是2583716229@qq.com。

**Comment by MaxMKMax** [comment]
@Joah618 您好, 那是因为系统问题. 我们可以重新帮您升级. 请您也提供我支付单据.

**Comment by Joah618** [comment]
你好，这是支付单据

**Reply by Joah618** [reply]
![D39A3841-9897-4B91-B3EB-EE9E611D4A24](https://github.com/user-attachments/assets/3c578cba-6d34-43ff-8724-d75c954b4e8e)
